LuaUI/Widgets/cmd_hold_ctrl_to_place_and_morph.lua

```diff
@@ -0,0 +1,126 @@
+local widgetName = "Hold Ctrl during placement to morph";
+
+function widget:GetInfo()
+	return {
+		name      = widgetName,
```

Comment:
No need for an extra var.
```suggestion
function widget:GetInfo()
	return {
		name      = "Hold Ctrl during placement to morph",
```

Reply:
This was intended as a prefix for debug echos, but not actually used.